test/update_schema.ps1

```diff
@@ -0,0 +1,72 @@
+<#
```

Comment:
We should put this in build tools or part of korebuild eventually. Anywhere we use IIS tests will fail.